about/list_of_features.rst

```diff
@@ -95,14 +95,14 @@ Editor
 Rendering
 ---------
 
-3 rendering *methods* (running over 2 rendering *drivers*) are available:
-
-- **Forward+**, running over Vulkan 1.0 (with optional Vulkan 1.1 and 1.2
-  features). The most advanced graphics backend, suited for desktop platforms
-  only. Used by default on desktop platforms.
-- **Forward Mobile**, running over Vulkan 1.0 (with optional Vulkan 1.1 and 1.2
-  features). Less features, but renders simple scenes faster. Suited for mobile
-  and desktop platforms. Used by default on mobile platforms.
+3 rendering *methods* (running over 4 rendering *drivers*) are available:
+
+- **Forward+**, running over Vulkan / Direct3D 12 / Metal. The most advanced
+  graphics backend, suited for desktop platforms only. Used by default on
+  desktop platforms.
+- **Forward Mobile**, running over Vulkan / Direct3D 12 / Metal. Less features,
+  but renders simple scenes faster. Suited for mobile and desktop platforms.
+  Used by default on mobile platforms.
```

Comment:
```suggestion
Godot 4 includes three renderers:

- **Forward+**. The most advanced renderer, suited for desktop platforms only.
  Used by default on desktop platforms. This renderer uses **Vulkan**, **Direct3D 12**,
  or **Metal** as the rendering driver, and it uses the **RenderingDevice** backend.
- **Mobile**. Fewer features, but renders simple scenes faster. Suited for mobile
  and desktop platforms. Used by default on mobile platforms. This renderer uses
  **Vulkan**, **Direct3D 12**, or **Metal** as the rendering driver, and it uses
  the **RenderingDevice** backend.
```
This is what I currently have for this section in https://github.com/godotengine/godot-docs/pull/10183. It's word-for-word the same Renderers page summaryin that PR, too.